### DISCUSSION TITLE
update:modelValue vs update:model-value

### DISCUSSION THREAD

**Discussion by sumerokr** [vuejs/devtools-v6#2111] [General]
Hi there. First of all, thanks for the beloved extension!
I want to discuss the way `defineEmits` should be declared.
According to Vue docs, it should look like `defineEmits(['update:modelValue'])`. But when you do so, devtools show warning:
![image](https://github.com/vuejs/devtools/assets/1766126/9828177b-8bdb-42dc-b16e-4da9dfe32198)

Switching to `defineEmits(['update:model-value'])` helps to eliminate this warning, but the fact that it goes against default documented approach cab be confusing, especially to people how are new to Vue.

**Comment by Azurewarth0920** [comment]
This issue will be fixed with https://github.com/vuejs/devtools/pull/2051
